### PR TITLE
Update extension for phpspec ~3.0

### PR DESCRIPTION
**Pull request by jameshalsall**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -20,10 +20,10 @@
         "bin-dir" : "bin"
     },
     "require": {
-        "phpspec/phpspec": "~2.0,<=2.5"
+        "phpspec/phpspec": "~3.0"
     },
     "require-dev": {
         "behat/behat": "~3.0",
-        "symfony/filesystem": "~2.3"
+        "symfony/filesystem": "~3.0"
```
Comment (ciaranmcnulty) [inline composer.json]: Why?

**Comment by ciaranmcnulty** [comment]
Can you change the README too please to reflect the new config format?

File: composer.json (hunk 1 above)
Comment (jameshalsall) [inline composer.json]: `require-dev` is only used if it is in the root `composer.json` so it won't have any impact on projects using this extension (https://getcomposer.org/doc/04-schema.md#require-dev)

**Comment by jameshalsall** [comment]
Updated README, I've removed the version number in the `composer require` instructions too as the default behaviour is to use the latest stable

File: composer.json (hunk 1 above)
Comment (ciaranmcnulty) [inline composer.json]: Ah, of course it's in -dev

**Comment by ciaranmcnulty** [comment]
Thanks @jameshalsall !

**Pull request merged**